source/_integrations/netatmo.markdown

```diff
@@ -30,45 +30,27 @@ To enable the Netatmo component, add the following lines to your `configuration.
 ```yaml
 # Example configuration.yaml entry
 netatmo:
-  api_key: YOUR_CLIENT_ID
-  secret_key: YOUR_CLIENT_SECRET
-  username: YOUR_USERNAME
-  password: YOUR_PASSWORD
+  client_id: YOUR_CLIENT_ID
+  client_secret: YOUR_CLIENT_SECRET
 ```
 
 {% configuration %}
-api_key:
+client_id:
   description: The `client id` from your Netatmo app.
   required: true
   type: string
-secret_key:
+client_secret:
   description: The `client secret` from your Netatmo app.
   required: true
   type: integer
```

Comment:
I know I'm late to the party, but I'm pretty confused from the docs.

How can this ever be an integer? It's always be a `string`. It's even a string in the provided screenshot?

Reply:
@rugaard well spotted. Thanks. Do you mind fixing it and opening a PR?

Reply:
Done  https://github.com/home-assistant/home-assistant.io/pull/12009  👍

Reply:
Great. Thanks!